lib/jason.ex

```diff
@@ -6,7 +6,7 @@ defmodule Jason do
   alias Jason.{Encode, Decoder, DecodeError, EncodeError, Formatter}
 
   @type escape :: :json | :unicode_safe | :html_safe | :javascript_safe
-  @type maps :: :naive | :strict
+  @type maps :: :naive | :strict | :camel_case
```

Comment:
We have added the `:camel_case` in the `:maps` option, but what if I want to encode with `:strict` and `:camel_case`!
What I'm saying is that the `:maps` option is not related with the formatting. Maybe we should just add it to the `:encode_opt` options... because there is already there the `:pretty` flag that affects the formatting.

@sadesyllas can you please also give your opinion on this?

Reply:
As we talked about it last time, as long as you do **not** use exactly the same keys, using `:camel_case` and `:strict` is possible and will encode the map keys accordingly. *But*, Jason is unable to detect that the keys `:one_two` and `"one_two_"` are the same because of its implementation of `:strict`. So, as long as you knowingly do not use same map keys, you will be fine using both encode options.